### PR TITLE
Get-EntraFeatureRolloutPolicy

### PR DESCRIPTION
**Pull request by v-akarke**
Updated the doc and tested all the examples; everything is working as expected.

### CODE REVIEW

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraFeatureRolloutPolicy.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+```powershell
+Get-EntraFeatureRolloutPolicy 
+ -Id <String> 
+ [<CommonParameters>]
+```
+
+## Description
+
+The `Get-EntraFeatureRolloutPolicy` cmdlet allows an admin to get the policy for cloud authentication rollout (users moving from federation to cloud auth) in Microsoft Entra ID.
+This policy is in the form of one or two FeatureRolloutPolicy objects holding groups that are assigned for cloud auth (Pass-through auth or Password hash-sync) and groups that are assigned for Seamless single sign-on (feature on top of PTA or PHS).
```
Comment (SteveMutungi254) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraFeatureRolloutPolicy.md:54]:
```suggestion

This policy is in the form of one or two FeatureRolloutPolicy objects holding groups that are assigned for cloud auth (Pass-through auth or Password hash-sync) and groups that are assigned for Seamless single sign-on.
```

**Review by SteveMutungi254 — DISMISSED** [review]
LGTM.

File: module/Entra/AdditionalFunctions/Get-EntraFeatureRolloutPolicy.ps1 [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    [CmdletBinding(DefaultParameterSetName = 'GetQuery')]
+    param (
+    [Parameter(ParameterSetName = "GetById", Mandatory = $true, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [System.String] $Id,
+    [Parameter(ParameterSetName = "GetVague", ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [System.String] $SearchString,
+    [Parameter(ParameterSetName = "GetQuery", ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [System.String] $Filter,
+    [Parameter(Mandatory = $false, ValueFromPipeline = $false, ValueFromPipelineByPropertyName = $true)]
+    [System.String[]] $Property
```
Comment (KenitoInc) [inline module/Entra/AdditionalFunctions/Get-EntraFeatureRolloutPolicy.ps1:14]: Handle this Param below

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraFeatureRolloutPolicy.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+# Get-EntraFeatureRolloutPolicy
+
+## Synopsis
+
+Gets the policy for cloud authentication roll-out in Microsoft Entra ID.
+
+## Syntax
+
+### GetQuery (Default)
```
Comment (KenitoInc) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraFeatureRolloutPolicy.md:27]: Add `Property`

File: module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraFeatureRolloutPolicy.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+
+Id                                   Description            DisplayName Feature            IsAppliedToOrganization IsEnabled
+--                                   -----------            ----------- -------            ----------------------- ---------
+aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb  Feature-Rollout-Policy change      emailAsAlternateId False                   False
+
+```
+
+This command retrieves the policy for cloud authentication roll-out policy in Microsoft Entra ID.
+
+## Parameters
```
Comment (KenitoInc) [inline module/docs/entra-powershell-v1.0/Microsoft.Graph.Entra/Get-EntraFeatureRolloutPolicy.md:126]: Add `Property`

File: module/Entra/AdditionalFunctions/Get-EntraFeatureRolloutPolicy.ps1 (hunk 2 above)
Comment (v-akarke) [reply module/Entra/AdditionalFunctions/Get-EntraFeatureRolloutPolicy.ps1:14]: Updated!

**Review by SteveMutungi254 — DISMISSED** [review]
LGTM.

File: module/Entra/AdditionalFunctions/Get-EntraFeatureRolloutPolicy.ps1 [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+            foreach ($response in $data) {
+                $userType = New-Object Microsoft.Graph.PowerShell.Models.MicrosoftGraphFeatureRolloutPolicy
+                $response.PSObject.Properties | ForEach-Object {
+                    $propertyName = $_.Name
+                    $propertyValue = $_.Value
+                    $userType | Add-Member -MemberType NoteProperty -Name $propertyName -Value $propertyValue -Force
+                }
+                $userList += $userType
+            }
+            if($null -ne $PSBoundParameters["Property"]){
```
Comment (KenitoInc) [inline module/Entra/AdditionalFunctions/Get-EntraFeatureRolloutPolicy.ps1:65]: We should not be using `Select-Object` after extracting the results. 
The correct approach should be adding a `$select` query option.
image

Sample implementation https://github.com/microsoftgraph/entra-powershell/blob/642a35c494d9a3fbe470f1da559c943472596cde/module/Entra/AdditionalFunctions/Get-EntraObjectSetting.ps1#L26-#L32

**Review by KenitoInc — CHANGES_REQUESTED** [review]

File: module/Entra/AdditionalFunctions/Get-EntraFeatureRolloutPolicy.ps1 (hunk 5 above)
Comment (v-akarke) [reply module/Entra/AdditionalFunctions/Get-EntraFeatureRolloutPolicy.ps1]: Updated the customization!

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**